xml/obs_moderation.xml

```diff
@@ -41,11 +41,43 @@
       </listitem>
       <listitem>
         <para>
-          Any other outcome of misconduct, according to our
-          <link xlink:href="https://build.opensuse.org/code_of_conduct">Code of Conduct</link>.
+          Any other outcome of misconduct, according to the <xref linkend="code_of_conduct"/>.
```

Comment:
I think you can drop this whole section....

Reply:
Are you talking about the last item on the list or the whole section `What Is Problematic Content for OBS`?  (leaving the Code of Conduct part, of course)

Reply:
the whole sect1